beacon_node/eth1/src/service.rs

```diff
@@ -142,7 +157,27 @@ impl EndpointsCache {
                             &[endpoint_str],
                         );
                         match func(&endpoint.endpoint).await {
-                            Ok(t) => Ok(t),
+                            Ok(t) => {
+                                let (prev_url, changed) =
+                                    self.update_cur_endpoint_url(endpoint.endpoint.clone());
+                                if changed {
+                                    info!(
```

Comment:
This is not quite the log format I was referring to. I've added an example of what I was trying to communicate in https://github.com/paulhauner/lighthouse/commit/d917c5d7650280e02f47380fd084528d1a45a0ac.